api/location.go

```diff
@@ -14,6 +14,7 @@ const (
 	LayoutCountry                             = "Country"
 	LayoutUSCityStateInternationalCity        = "US City, State, International city"
 	LayoutUSCityStateInternationalCityCountry = "US City, State, International city country"
+	LayoutState                               = "State"
```

Comment:
I don't see a `.go` file that addresses how this new field gets persisted. Does that work auto-magically? Have you done a smoke test where you set the state, log out, open new incognito browser, log in, and see data?

Reply:
Yeah I've tested it and it persists, I followed the same format as the other location layouts so I suspect it's inheriting some of the existing functionality.